exception from lineCount() if no traces for a particular module

when calling lineCount() at the end of a single block, there's no guarantee that all modules would have lineCounts > 0 however this is assumed with the current code 

example stack trace -

```
Caused by: java.lang.ArrayIndexOutOfBoundsException: Array index out of range: 0
	at java.base/java.util.Vector.get(Vector.java:750)
	at net.consensys.linea.zktracer.container.stacked.list.StackedList$StackedListIterator.list(StackedList.java:232)
	at net.consensys.linea.zktracer.container.stacked.list.StackedList$StackedListIterator.hasNext(StackedList.java:258)
	at net.consensys.linea.zktracer.module.rlp_txn.RlpTxn.lineCount(RlpTxn.java:1361)
	at net.consensys.linea.zktracer.ZkTracer.lambda$getModulesLineCount$0(ZkTracer.java:156)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at net.consensys.linea.zktracer.ZkTracer.getModulesLineCount(ZkTracer.java:156)
	at net.consensys.linea.zktracer.ZkTracer.traceEndBlock(ZkTracer.java:93)
	at org.hyperledger.besu.ethereum.blockcreation.AbstractBlockCreator.createBlock(AbstractBlockCreator.java:286)
	... 7 common frames omitted
```

--- a/arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/list/StackedList.java
+++ b/arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/list/StackedList.java
@@ -27,7 +27,7 @@ import net.consensys.linea.zktracer.container.StackedContainer;
 import org.jetbrains.annotations.NotNull;
 
 /**
- * Implements a system of nested lists behaving as a single on, where the current context
+ * Implements a system of nested lists behaving as a single one, where the current context
  * modification can transparently be dropped.
  *
  * @param <E> the type of elements stored in the list

--- a/arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/list/StackedList.java
+++ b/arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/list/StackedList.java
@@ -229,6 +229,9 @@ public class StackedList<E> implements List<E>, StackedContainer {
     }
 
     private List<F> list() {
+      if (sl.lists.size() == 0) {
+        return List.of();
+      }
       return this.sl.lists.get(this.head);
     }
 

--- a/arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/set/StackedSet.java
+++ b/arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/set/StackedSet.java
@@ -25,7 +25,7 @@ import net.consensys.linea.zktracer.container.StackedContainer;
 import org.jetbrains.annotations.NotNull;
 
 /**
- * Implements a system of nested sets behaving as a single on, where the current context
+ * Implements a system of nested sets behaving as a single one, where the current context
  * modification can transparently be dropped.
  *
  * @param <E> the type of elements stored in the set

